Add the “Data Fetching” exercise solution

chasenlehara opened the pull request:
For this exercise, I think we will start with almost all of this code, minus the `aria`, `role`, and `sr-only` bits. I will write the exercise text when I make the problem folder.

Solution: https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/accessible-data-fetching/incorporating-accessibility-into-your-dev-process/data-fetching/solution?file=src/App.tsx

CMLabay commented on incorporating-accessibility-into-your-dev-process/data-fetching/solution/src/App.tsx:
```diff
@@ -0,0 +1,107 @@
+                  <th scope="col">Occupation</th>
+                </tr>
+              </thead>
+              <tbody aria-busy={loading ? "true" : "false"}>
```
I think this can be just `<tbody aria-busy={loading}>`

chasenlehara replied:
Yup, thanks for catching this!

CMLabay reviewed (approved):
Looks good, just left one comment

chasenlehara committed:
Add the “Data Fetching” exercise solution

For this exercise, I think we will start with almost all of this code, minus the `aria`, `role`, and `sr-only` bits. I will write the exercise text when I make the problem folder.
